docker-compose.yml

```
@@ -0,0 +1,20 @@
+services:
+  db:
+    image: bitnami/postgresql:15
```

Could we use version 17 to keep it inline with our production use?
```suggestion
    image: bitnami/postgresql:17
```

---

15 is more like LTS right now, it wont break anything, the 17 might in some case